server/conf/application.conf

```diff
@@ -145,8 +145,10 @@ addressIndex {
         pafWelshDoubleDependentLocalityBoost=0.3
         pafWelshDoubleDependentLocalityBoost=${?ONS_AI_API_QUERY_LOCALITY_PAF_WELSH_DOUBLE_DEPENDENT_LOCALITY_BOOST}
       }
-      disMaxTieBreaker=0.5
-      disMaxTieBreaker=${?ONS_AI_API_QUERY_TIE_BREAKER_BOOST}
+      excludingDisMaxTieBreaker=0.0
+      excludingDisMaxTieBreaker=${?ONS_AI_API_QUERY_EXCLUDING_TIE_BREAKER_BOOST}
+      includingDisMaxTieBreaker=0.5
```

Comment:
I am not sure what this value should be. We can start with 0.5 and test using the env variable to find an optimal value.